src/Item_Project.php

```diff
@@ -208,8 +208,7 @@ public static function showForProject(Project $project): bool
         }
         if ($totalnb > 0) {
             echo "<tr class='tab_bg_2'>";
-            echo "<td class='center' colspan='2'>" .
-               (($totalnb > 0) ? sprintf(__s('%1$s = %2$s'), __('Total'), $totalnb) : "&nbsp;");
+            echo "<td class='center' colspan='2'>" . sprintf(__s('%1$s = %2$s'), __('Total'), $totalnb);
```

Comment:
```suggestion
            echo "<td class='center' colspan='2'>" . sprintf(__s('%1$s = %2$s'), __s('Total'), $totalnb);
```

Reply:
Is this needed ? The content of sprintf is already using `__s`.